authors/apps/authentication/backends.py

```diff
@@ -8,5 +8,6 @@
 
 """Configure JWT Here"""
 class JWTAuthentication:
-    pass
+    def authenticate(self, another):
```

Comment:
Did you do the rebase??

Reply:
Yes, i did.

Reply:
this class should not be appearing then.

Reply:
why?

Reply:
Once you do a rebase, those changes shouldn't be appearing as yours.

Reply:
Am the one who made that change for documentation to work because it was raising an error hinting that authenticate method is missing.